server/src/subscriber/subscriber.controller.ts

```diff
@@ -122,4 +122,44 @@ export class SubscriberController {
 
     response.send(updatedContact);
   }
+
+  @Get("/subscribers/email/:email")
+  async getEmail(@Param() params, @Res() response) {
+    if (!validateEmail(params.email)) {
+      response.status(400).send({
+        error: "Invalid email address."
+      })
+      return;
+    }
+
+    const existingUser = await this.subscriberService.findByEmail(params.email);
+    if (existingUser.code === 404) {
+      response.status(404).send({
+        error: "User not found."
+      })
+      return;
+    }
+
+    const userExistsInCurrentList = existingUser['1'].result[params.email].contact.list_ids.includes(this.list);
+    const existingUserListConfirmed = existingUser['1'].result[params.email].contact.custom_fields[`zap_${this.sendgridEnvironment}_confirmed`];
+
+    if (userExistsInCurrentList && existingUserListConfirmed) {
+      response.status(201).send({
```

Comment:
What about instead of returning a message that we'd have to do string comparison on in the front end, we return data about the user? (cc @dhochbaum-dcp )
```
{
  confirmed: true
}
```

Reply:
Can we do both? Like 
```
{
    confirmed: true
    message: "User already subscribed." 
}
```
so that we can be consistent with our API responses and having human readable language to fall back on ?